blitz_api/serializers.py

```diff
@@ -8,7 +8,7 @@
 
 class AuthCustomTokenSerializer(serializers.Serializer):
     """
-
+    Verifies if the provided login is an email or username
```

Comment:
It's not just that, it check if the login and password can authenticate a user too!

In fact it's the token serializer, it could also be used to return a token in a standard form (currently the view does it :cry: )